README.md

```diff
@@ -10,6 +10,8 @@
 
 loxs is a C++20 implementation of Bob Nystrom's toy language, [Lox](https://craftinginterpreters.com).
 
+Lox Git: [Lox Git](https://github.com/munificent/craftinginterpreters)
```

Comment:
Cool, but maybe add some more context please, like "Lox(link here) github repository(or link here) containing the markdown's of the text book and a full implementation of both interpreters" etcetc :)

Reply:
Sure, lets put this prettier eheh